CONTRIBUTING.md

```diff
@@ -168,36 +165,41 @@ This tag is for boolean properties associated with the test.
 
+#### es5id
+**es5id**: [es5-test-id]
+
+**Deprecated.**
+
+This tag identifies the section number from the portion of the ECMAScript 5.1 or ECMAScript 3 standard that is tested by this test. It was automatically generated for tests that were originally written for the ES5 (or earlier) version of the test suite and are now part of the ES6 version. You can use the es5id to discover the relevant portion of the ECMAScript standard by looking up the section number in previous publications of the specification:
+https://www.ecma-international.org/publications/standards/Ecma-262-arch.htm
+
+#### es6id
+**es6id**: [es6-test-id]
+
+**Deprecated.**
+
+This tag identifies the section number from the portion of the ES6 standard that is tested by this test _at the time the test was written_. The es6ids might not correspond to the correction section numbers in the ES6 (or later) specifcation because routine edits to the specification will change section numbering. For this reason, only the esid is required for new tests.
+
 ## Test Environment
 
 Each test case is run in a fresh JavaScript environment; in a browser, this will be a new `IFRAME`; for a console runner, this will be a new process.  The test harness code is loaded before the test is run.  The test harness defines the following helper functions:
 
 Function | Purpose
 ---------|--------
 Test262Error(message) | constructor for an error object that indicates a test failure
-$ERROR(message) | construct a Test262Error object and throw it
 $DONE(arg) | see Writing Asynchronous Tests, below
 assert(value, message) | throw a new Test262Error instance if the specified value is not strictly equal to the JavaScript `true` value; accepts an optional string message for use in creating the error
 assert.sameValue(actual, expected, message) | throw a new Test262Error instance if the first two arguments are not [the same value](https://tc39.github.io/ecma262/#sec-samevalue); accepts an optional string message for use in creating the error
 assert.notSameValue(actual, unexpected, message) | throw a new Test262Error instance if the first two arguments are [the same value](https://tc39.github.io/ecma262/#sec-samevalue); accepts an optional string message for use in creating the error
 assert.throws(expectedErrorConstructor, fn, message) | throw a new Test262Error instance if the provided function does not throw an error, or if the constructor of the value thrown does not match the provided constructor
 assert.throws.early(expectedErrorConstructor, code) | throw a new Test262Error instance if the provided code does not throw an early error, or if the constructor of the value thrown does not match the provided constructor. This assertion catches only errors that will be parsed through `Function(code)`.
+$ERROR(message) | construct a Test262Error object and throw it <br>**DEPRECATED** -- do not use new tests
```

Comment:
s/do not use new tests/do not use in new tests

or maybe tell to use one of the assertions or throw a `new Test262Error` instead?